packages/eas-cli/src/build/build.ts

```diff
@@ -274,7 +279,24 @@ async function uploadProjectAsync<TPlatform extends Platform>(
             completedMessage: (duration: string) => `Uploaded to EAS ${chalk.dim(duration)}`,
           })
         );
-        return bucketKey;
+
+        const projectMetadataFile = await makeProjectMetadataFileAsync(projectTarball.path);
+
+        const metadataLocation = await uploadFileAtPathToGCSAsync(
+          ctx.graphqlClient,
+          UploadSessionType.EasBuildGcsProjectSources,
+          projectMetadataFile.path,
+          createProgressTracker({
+            total: projectMetadataFile.size,
+            message: ratio =>
+              `Uploading metadata to EAS Build (${formatBytes(projectMetadataFile.size * ratio)} / ${formatBytes(
+                projectMetadataFile.size
+              )})`,
+            completedMessage: (duration: string) => `Uploaded to EAS ${chalk.dim(duration)}`,
+          })
+        );
```

Comment:
What does an example path to a metadata file look like?

How long do we want to keep this file around? What is the lifecycle rule for this bucket? 2hrs?

Reply:
Objects in `turtle-v2-projects` are stored for [up to 60 days](https://github.com/expo/universe/blob/6b9314d9b34c6af2f912bd7d544660a0c98df383/terraform/turtle_v2_gcs.tf#L100-L117), but usually they're deleted [right after `retryExpiresAt` by `buildReaper`](https://github.com/expo/turtle-v2/blob/69f901feebc8b3a47063dc62165bf387a5efb72f/src/services/turtle-api/src/storage/db.ts#L114-L124).

Reply:
@szdziedzic nope, it only affects `projectArchive` files

Reply:
@khamilowicz So metadata file will be kept around for 60 days (default bucket policy)?

Reply:
@szdziedzic right